### ISSUE TITLE
Option to auto set label when adding torrent

### ISSUE THREAD

**Issue by My-userid** [erickok/transdroid#360]
Hi there

Love your program. 
I would love to be able to specify a label to be auto applied when adding torrents. 

For now I add torrents in paused stage, apply label and then start the torrent. 

In rtorrent I have setup to move downloads to specific folders based on their label. 

Would be nice to have the last automation in Transdroid too.

**Comment by erickok** [comment]
While I cannot find a duplicate issue now, this has indeed been requested before. For the moment I decided not to introduce any dialog when adding a torrent (to set label, location or other settings before starting it). But perhaps in the future, who knows.

**Comment by vicelversa** [comment]
Would also like to see this as I use a similiar method of moving torrents around based on their label name.

In the meantime, @My-userid how do you add torrents to transdroid in a paused state?

**Comment by My-userid** [comment]
@vicelversa 

I use rTorrent and in .rtorrent.rc (config file for rtorrent) I added the following:

# Stop new torrent without a label
system.method.set_key = event.download.inserted_new,stop_tor,"branch=d.get_custom1=,d.start=,d.stop="

**Comment by ChickenDipper1** [comment]
Is there any progress on having a prompt to ask for labels and/or download DIR? Specifically for Qbittorrent.

**Comment by erickok** [comment]
No sorry, no progress, as I am not actively working on the app at the
moment (beyond the occasional bugfix).

**Comment by ChickenDipper1** [comment]
That's a real shame, is it likely to implemented any time in the near future as it would really complete the app for me, helps a lot with organising files.

**Comment by erickok** [comment]
Sorry, but I do not think so.
